application/templates/backup.yaml

```
@@ -0,0 +1,17 @@
+{{- if (.Values.backup).enabled }}
```

```suggestion
{{- if .Values.backup.enabled }}
```
And add default value file with `backup.enabled: false` and some other sane default values if you want.

---

Agreed @aslafy-z we should do that

---

Added. Resolving comment